db/migrate/20180328074226_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+
+      t.string :password_digest
+
+      t.text   :token
```

Comment:
name it, please, as `auth_token` or smth like that. More flexible, IMO
___
Stop, we decided not to save token in db. Is that ok?